modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/AndesContextInformationManager.java

```diff
@@ -185,7 +185,7 @@ public void deleteQueue(String queueName) throws AndesException {
         }
 
         //purge the queue cluster-wide
-        MessagingEngine.getInstance().purgeMessages(queueName, null, false,null);
+        MessagingEngine.getInstance().purgeMessages(queueName, null, false, 0l);
```

Comment:
this method signature/contract will change since we drop cassandra as a message store. therefore ok for now.